bylaws/content/council.tex

```diff
@@ -41,8 +41,8 @@ \subsection{Representative Allocation}
 the purposes of determining if a student is in a program, minors and options are
 not counted unless explicitly mentioned. A student may be
 counted in more than one constituency. Notwithstanding anything else in this
-section, if a student is counted in the First Year constituency, then they are
-counted in no other constituency.
+section, in the Fall and Winter terms, if a student is counted in the First Year constituency, 
+then they are counted in no other constituency.
```

Comment:
Add: In Spring terms, if a student would be counted in the First Year constituency then they are instead counted in the At-large constituency.